docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -151,40 +151,40 @@ The GNA plugin supports the following configuration parameters for multithreadin
 
-> **NOTE:** Due to limitation of the Intel&reg; GNA backend library, heterogenous support is limited to cases where in the resulted sliced graph, only one subgraph is scheduled to run on GNA\_HW or GNA\_SW devices.
+> **NOTE:** Due to limitation of the Intel® GNA backend library, heterogenous support is limited to cases where in the resulted sliced graph, only one subgraph is scheduled to run on GNA\_HW or GNA\_SW devices.
 
-## Recovery from interruption by high-priority Windows audio processes\*
+## Recovery from Interruption by High-Priority Windows Audio Processes\*
 
-As noted in the introduction, GNA is designed for real-time workloads such as noise reduction.
+GNA is designed for real-time workloads such as noise reduction.
 For such workloads, processing should be time constrained, otherwise extra delays may cause undesired effects such as
-audio "glitches". To make sure that processing can satisfy real time requirements, the GNA driver provides a QoS
-(Quality of Service) mechanism which interrupts requests that might cause high-priority Windows audio processes to miss
-schedule, thereby causing long running GNA tasks to terminate early.
+*audio glitches*. To make sure that processing can satisfy real-time requirements, the GNA driver provides a Quality of Service
+(QoS) mechanism, which interrupts requests that might cause high-priority Windows audio processes to miss
+the schedule, thereby causing long running GNA tasks to terminate early.
 
 Applications should be prepared for this situation.
-If an inference (in `GNA_HW` mode) cannot be executed because of such an interruption, then `InferRequest::Wait()` will return status code
-`StatusCode::INFER_NOT_STARTED` (note that it will be changed to a more meaningful status code in future releases).
+If an inference in the `GNA_HW` mode cannot be executed because of such an interruption, then `InferRequest::Wait()` returns status code
+`StatusCode::INFER_NOT_STARTED`. In future releases, it will be changed to a more meaningful status code.
 
-Any application working with GNA must properly react if it receives this code. Various strategies are possible.
-One of the options is to immediately switch to GNA SW emulation mode:
+Any application working with GNA must properly react to this code.
+One of the strategies to adapt an application:
 
+1. Immediately switch to the GNA_SW emulation mode:
 ```cpp
 std::map<std::string, Parameter> newConfig;
 newConfig[GNAConfigParams::KEY_GNA_DEVICE_MODE] = Parameter("GNA_SW_EXACT");
 executableNet.SetConfig(newConfig);
 
 ```
-
-then resubmit and switch back to GNA_HW after some time hoping that the competing application has finished.
+2. Resubmit and switch back to GNA_HW when the competing application has finished.
```

Comment:
This completely changes the meaning. We don't know when the competing application has finished. We can only try again after some delay, but there is no guarantee that it will work out.